include/vcl/Image.h

```diff
@@ -213,7 +218,7 @@ namespace VCL {
          *  @see OpenCV documentation for imencode for more details
          */
         std::vector<unsigned char> get_encoded_image(VCL::Image::Format format,
-                const std::vector<int>& params=std::vector<int>()) const;
+                const std::vector<int>& params=std::vector<int>());
```

Comment:
Mainly, if those commits have been merged here, then the removing API layers isn;t the only change here. It also introduced changes to copying methods. So the name / commit messages should reflect that. and preferably be separate. It is hard to isolate problems when commits are combined. I must admit, it is entirely possible that I am seeing a condensed version due to github GUI. So if that is the case, ignore my comment. But if you have combined the shallow/deep copy changes with APi layers into one commit, please segregate them.

Reply:
All these changes are done on top of the changes that has been merged by Luis (commit 5184192). Hence, the changes are not related.

In this specific instance, 'const' is removed because 'get_encoded_image' modifies the members of the Image class now that the ImageData class is gone (previously, it was const because no member of the Image class was modified)